serverless/serverless-changelog.asciidoc

```diff
@@ -9,6 +9,67 @@ For serverless changes in Cloud Console, refer to https://www.elastic.co/guide/e
 [[serverless-changelog-12032024]]
 == December 3, 2024
 
+[discrete]
+[[known-issues-12032024]]
+=== Known issues
+
+// tag::known-issue[201820]
+[discrete]
+.In {sec-serverless}, the **Exceptions** tab won't properly load if exceptions contain comments with newline characters (`\n`)  
+[%collapsible]
+====
+*Details* +
+On December 5, 2024, it was discovered that the **Exceptions** tab will not load properly if any exceptions contain comments with newline characters (`\n`). This issue occurs when you upgrade to 8.16.0 or later. 
+
+*Workaround* + 
+
+For custom rules:
+
+. From your {sec-serverless} project, <<import-export-rules-ui,export>> the rule or rules with the affected exception lists. 
+. Modify the `.ndjson` file so `comments` no longer contain newline characters.
+. Return to your project and <<import-export-rules-ui,re-import>> the rules. Make sure to select the **Overwrite existing exception lists with conflicting "list_id"** option.
+
+For prebuilt rules: 
+
+. Fetch the affected exception list ID or IDs that are associated with the rule. 
+.. Find the affected rule's ID (`id`). From your {sec-serverless} project, open the rule's details page, go to the page URL, and copy the string at the end. For example, the URL http://host.name/app/security/rules/id/167a5f6f-2148-4792-8226-b5e7a58ef46e contains the `id` `167a5f6f-2148-4792-8226-b5e7a58ef46e` at the end.
+.. Use the `id` to fetch the rule's details using the {api-kibana}/operation/operation-readrule[Retrieve a detection rule API]. Here is an example request that includes the rule ID:
```

Comment:
```suggestion
.. Use the `id` to fetch the rule's details using the {api-kibana-serverless}/operation/operation-readrule[Retrieve a detection rule API]. Here is an example request that includes the rule ID:
```

Reply:
Dependant on https://github.com/elastic/docs/pull/3137.

Reply:
https://github.com/elastic/docs/pull/3137 is now merged. 🚢

Reply:
Added in [7bdffd5](https://github.com/elastic/docs-content/pull/233/commits/7bdffd5c5ed1b5dc5d61d4ead29c55ab9329de43).